examples/package.json

```diff
@@ -19,25 +20,24 @@
     "ts:sdk:create_log": "npm run run:ts -- ./src/sdk/create-log/example.ts",
     "ts:sdk:annotate_log": "npm run run:ts -- ./src/sdk/annotate-log/example.ts",
     "ts:sdk:rag_retrieval": "npm run run:ts -- ./src/sdk/rag-retrieval/example.ts",
-
     "run:py": "PYTHONPATH=src dotenv -e .env -- .venv/bin/python",
     "py:sdk:get_prompt": "npm run run:py -- ./src/sdk/get-prompt/example.py",
     "py:sdk:get_all_prompts": "npm run run:py -- ./src/sdk/get-all-prompts/example.py",
+    "py:sdk:get_versions": "npm run run:py -- ./src/sdk/get-versions/example.py",
     "py:sdk:create_prompt": "npm run run:py -- ./src/sdk/create-prompt/example.py",
     "py:sdk:run_prompt": "npm run run:py -- ./src/sdk/run-prompt/example.py",
     "py:sdk:run_prompt_with_tools": "npm run run:py -- ./src/sdk/run-prompt-with-tools/example.py",
     "py:sdk:render_chain": "npm run run:py -- ./src/sdk/render-chain/example.py",
     "py:sdk:pause_tools": "npm run run:py -- ./src/sdk/pause-tools/example.py",
     "py:sdk:create_log": "npm run run:py -- ./src/sdk/create-log/example.py",
     "py:sdk:annotate_log": "npm run run:py -- ./src/sdk/annotate-log/example.py",
-
     "ts:cases:customer_support": "npm run run:ts -- ./src/cases/customer-support-email/example.ts",
     "ts:cases:content_moderation": "npm run run:ts -- ./src/cases/content-moderation-system/example.ts"
   },
   "dependencies": {
     "@ai-sdk/openai": "^1.0.8",
     "@anthropic-ai/sdk": "^0.32.1",
-    "@latitude-data/sdk": "^4.1.2",
+    "@latitude-data/sdk": "5.0.0-beta.2",
```

Comment:
Is beta.3 no?

Reply:
I though the release happen on merge in the CI

Reply:
it does but i cannot test it before it has been released